emsdk.py

```diff
@@ -1129,6 +1130,64 @@ def build_llvm_tool(tool):
   return success
 
 
+# LLVM git source tree migrated to a single repository instead of multiple ones, build_llvm_monorepo() builds via that repository structure
+def build_llvm_monorepo(tool):
+  debug_print('build_llvm_monorepo(' + str(tool) + ')')
+  fastcomp_root = tool.installation_path()
+  fastcomp_src_root = os.path.join(fastcomp_root, 'src')
+  success = git_clone_checkout_and_pull(tool.download_url(), fastcomp_src_root, tool.git_branch)
+  if not success:
+    return False
+
+  build_dir = fastcomp_build_dir(tool)
+  build_root = os.path.join(fastcomp_root, build_dir)
+
+  build_type = decide_cmake_build_type(tool)
+
+  # Configure
+  global BUILD_FOR_TESTING, ENABLE_LLVM_ASSERTIONS
+  tests_arg = 'ON' if BUILD_FOR_TESTING else 'OFF'
+
+  enable_assertions = ENABLE_LLVM_ASSERTIONS.lower() == 'on' or (ENABLE_LLVM_ASSERTIONS == 'auto' and build_type.lower() != 'release' and build_type.lower() != 'minsizerel')
+
+  if ARCH == 'x86' or ARCH == 'x86_64':
+    targets_to_build = 'WebAssembly;X86'
+  elif ARCH == 'arm':
+    targets_to_build = 'WebAssembly;ARM'
```

Comment:
Does emsdk support arm?

Reply:
Apparently so, that is what we have on fastcomp build also.